internal/driver/glfw/window.go

```diff
@@ -130,6 +130,10 @@ func (w *window) Show() {
 
 func (w *window) doShow() {
 	if w.view() != nil {
+		runOnDraw(w, func() {
+			w.driver.repaintWindow(w)
+		})
```

Comment:
Actually, I had another look and wonder if it perhaps would be clearer to put this inside `doShowAgain()` instead?